tests/parameters/test_image.py

```diff
@@ -245,7 +245,7 @@ async def test_image_handler(respx_mock, image_handler: ImageParameterHandler):
     assert image_handler.initialized
     image_0 = image_handler["0"]
     assert image_0.enabled is True
-    assert image_0.name == "View Area 1"
+    assert image_0.name == "Fußgängerüberweg"
```

Comment:
😆 Im sad you can't use furzekanone here

Reply:
Sad to read this :-), whatever please you that can't be encoded with utf-8 is fine !